android/src/main/java/com/opentokreactnative/OTSubscriberViewManager.java

```diff
@@ -26,8 +26,9 @@ protected OTSubscriberLayout createViewInstance(ThemedReactContext reactContext)
 
     @ReactProp(name = "streamId")
     public void setStreamId(OTSubscriberLayout view, String streamId) {
-
+        if(streamId != null) {
```

Comment:
I think the indentation of this if block looks wrong.

Reply:
the indentation is correct

Reply:
Hi I have formatted as you said please check